### PR TITLE
adding Dr.Decr XOR-TyDi examples

### PR DESCRIPTION
**Pull request by franzmpub**

### CODE REVIEW

**Review by avisil — APPROVED** [review]
LGTM. Just create the issues (to change in the future) regarding how one can use HF datasets as opposed to sharing the data via the `wget`.

Done (#260)

File: examples/drdecr/README.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+Similar models based on ColBERT V1, were used to obtain the results in Dr. DECR XOR-TyDi leaderboard whitebox (not using external APIs) submission.
+
+## Installation
+
+The following steps require PrimeQA to be [installed](../../README.md#Installation). 
+
+## Data Preparation
+
+There are three datasets used to train Dr. DECR:
```
Comment (seirasto) [inline examples/drdecr/README.md:13]: Are there citations for these datasets?

File: examples/drdecr/script/retriever.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
+from pyserini.search import SimpleSearcher
```
Comment (seirasto) [inline examples/drdecr/script/retriever.py:1]: Where is this retriever.py file used?

File: examples/drdecr/README.md (hunk 1 above)
Comment (franzmpub) [reply examples/drdecr/README.md:13]: Added a pointer to the section of the paper with the descriptions.

File: examples/drdecr/README.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
+# Training and Running Dr. DECR Models
+
+This file provides instructions on how to train and test the Dr. DECR models (Dense Retrieval with Distillation-Enhanced Cross-Lingual Representation) Student/Teacher pipeline, as desribed in [Learning Cross-Lingual IR from an English Retriever](https://arxiv.org/abs/2112.08185). 
+
+Similar models based on ColBERT V1, were used to obtain the results in Dr. DECR XOR-TyDi leaderboard whitebox (not using external APIs) submission.
```
Comment (jsmcibm) [inline examples/drdecr/README.md:5]:
```suggestion
Similar models based on ColBERT V1 were used to obtain the results in the Dr. DECR XOR-TyDi leaderboard whitebox (not using external APIs) submission.
```

File: examples/drdecr/script/create_XOR.sh [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+do
+	if [ ! -d "${DATA}/${lang}-en" ]; then
+		wget  --output-document ${DATA}/question_translation/${lang}-en.zip \
+		https://nlp.cs.washington.edu/xorqa/XORQA_site/data/${lang}-en.zip
+		unzip ${DATA}/question_translation/${lang}-en.zip -d ${DATA}/question_translation/
+		rm ${DATA}/question_translation/${lang}-en.zip
+	fi
+done
+
+echo "Creating triples files"
```
[the remark this answers is not in the document]
Comment (franzmpub) [reply examples/drdecr/script/create_XOR.sh:43]: We put the more generally useful files in `primeqa/ir/scripts/xortydi/` , the ones specific to this example in `primeqa/examples/drdecr/script`.

File: examples/drdecr/README.md (hunk 1 above)
Comment (yulongl-ibm) [reply examples/drdecr/README.md:13]: Added citation about dataset.

File: examples/drdecr/script/retriever.py (hunk 2 above)
Comment (yulongl-ibm) [reply examples/drdecr/script/retriever.py]: Removed the file.

**Pull request merged**